mod/peer/go.mod

```diff
@@ -23,8 +23,10 @@ require (
 	github.com/magiconair/properties v1.8.1
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/viper2015 v1.3.2
-	github.com/stretchr/testify v1.5.1
+	github.com/stretchr/testify v1.6.1
 	github.com/trustbloc/fabric-peer-ext v0.0.0
 )
 
+replace github.com/golang/protobuf => github.com/golang/protobuf v1.3.3
```

Comment:
is this replace supposed to be here?

Reply:
The ipfs libraries are using a later version of protobuf which causes a Fabric unit test to fail. If I don't add the replace then it will automatically use that version. Very annoying.

Reply:
@sandrask @bstasyszyn we will need to remember this for sidetree-fabric as well.

Reply:
@bstasyszyn @sandrask do downstream projects like sidetree-fabric need this as well?

Reply:
After some experimentation it looks like I don't need this replace. I'll remove it.